Content.Server/Bible/Components/SummonableComponent.cs

```diff
@@ -31,6 +31,14 @@ public sealed partial class SummonableComponent : Component
         [DataField("requiresBibleUser")]
         public bool RequiresBibleUser = true;
 
+        // DeltaV - Begin Anomalite Limited Respawns
+        [DataField("lives")]
```

Comment:
Data field Lives in data definition SummonableComponent has an explicitly set tag that matches autogenerated tag
```suggestion
        [DataField]
```

Reply:
Ngl this one seems odd to me, since all the upstream changes seem to do the same, i assume its so if you change the field name you are fully aware you will have to change yml, but i suppose the field is technically optional

Reply:
> Ngl this one seems odd to me, since all the upstream changes seem to do the same, i assume its so if you change the field name you are fully aware you will have to change yml, but i suppose the field is technically optional

I believe setting the datafield name was mandatory at some point, but nowadays it defaults to the field name but with the first letter lowercased

Reply:
The explicit name was removed